### ISSUE TITLE
Calculation issue

### ISSUE THREAD

**Issue by LoveOfLava** [handsontable/handsontable-pro#157]
Hello

I don't know why the formula on B3 cell display  "#VALUE!" when the value on A3 is "nd"

<script async src="//jsfiddle.net/Hun7er/h4bsqwtr/41/embed/"></script>

Thanks
Regards

**Comment by AMBudnik** [comment]
Hi @LoveOfLava 

`B3` shows `#REF!`

The formula is a bit complicated

`=IF(OR(A3<E3,A3="nd"),"nd",IF(A3*(A1-A2)/A1<3*E3,"<"&ROUND(3*E3,3),A3*(A1-A2)/A1))`

You are refering to a cell `E3` that does not exist. If I add another column to create `E3` cell the `B3` becomes `#VALUE!`.

Can you tell me what are the requirements for this cell?

**Comment by LoveOfLava** [comment]
Hi,
The formula is located on the B3 cell

When I enter the value "nd" for A3 cell the formula display  "#VALUE!" 

https://jsfiddle.net/Hun7er/h4bsqwtr/44/

**Comment by AMBudnik** [comment]
It looks correct in the demo 

![image](https://user-images.githubusercontent.com/10757813/46804836-f2935b80-cd63-11e8-8451-79043be824c7.png)


even if I put `nd` manually in cell `A3` I do not get the error.

**Comment by LoveOfLava** [comment]
![image](https://user-images.githubusercontent.com/15012754/46804959-38e8ba80-cd64-11e8-9b90-8d9414f03149.png)

**Comment by AMBudnik** [comment]
What device and browser do you use?

**Comment by LoveOfLava** [comment]
Hello,

I think the link to demo is incorrect

Here is the right link

I use the latest chrome browser

https://jsfiddle.net/Hun7er/h4bsqwtr/48/

**Comment by AMBudnik** [comment]
OK, now I get the `#VALUE!`.
Let's check what is going on in the formula:
`'=IF(OR(A3<E3,A3="nd"),"nd",IF(A3*(A1-A2)/A1<3*E3,"<"&ROUND(3*E3,3),A3*(A1-A2)/A1))'`

the typical IF checks 
if `OR(A3<E3,A3="nd")` is true 
and if it gets `true` it will return `nd` 
otherwise you will get `IF(A3*(A1-A2)/A1<3*E3,"<"&ROUND(3*E3,3),A3*(A1-A2)/A1))'`

The first part return true, so we should get `nd` but as the second part produces and error `#VALUE!` whole formula gets corrupted. Especially the `A3*(A1-A2)` produces the `#VALUE!` error as you try to multiple text by a number `nd * (4-3)`

**Comment by LoveOfLava** [comment]
I think the behavior is not correct.
It must displaying "nd"

I tried to implement the formula on a Excel file and it displays "nd"

![image](https://user-images.githubusercontent.com/15012754/46806860-f8d80680-cd68-11e8-8445-ce0e6c0e6765.png)

**Comment by AMBudnik** [comment]
Yes, you are right. The formula should abadon the fact that the rest of the equasion is invalid as the IF statement turns our `true`. The result should be `nd`.

**Comment by LoveOfLava** [comment]
Hello,

You will solve the bug ?

**Comment by AMBudnik** [comment]
Yes, we should. However, I do not know if we won't just wait for the new engine to come.
